lib/elixir/lib/calendar.ex

```diff
@@ -721,10 +777,17 @@ defmodule Time do
   @spec to_erl(Calendar.time) :: :calendar.time
   def to_erl(time)
 
-  def to_erl(%{hour: hour, minute: minute, second: second}) do
+  def to_erl(%{hour: hour, minute: minute, second: second, calendar: Calendar.ISO}) do
```

Comment:
Instead of adding those two clauses everywhere, you can add it directly to convert:

```elixir
convert(%{calendar: calendar}, calendar), do: calendar
```